client/src/context/GameContext.tsx

```diff
@@ -47,12 +47,15 @@ interface ContextType {
   currStep: Step;
   stepIndex: number;
   level: number;
+  maxLevelSeen: number;
```

Comment:
Does `maxLevelSeen` have to be exposed? Are there components that require this value if `hasSeenLevel` is provided?